src/App.css

```diff
@@ -27,22 +27,103 @@
 .sub-container {
   position: relative;
   margin-left: 250px;
```

Comment:
I believe we don't need to add this padding. to the sub-container class as it creates the unnecessary op
![Screenshot 2022-10-24 at 18 11 47](https://user-images.githubusercontent.com/31113416/197561394-47646687-2a50-40b8-acb8-575b2b369512.png)